docs/form/index.html

```diff
@@ -16,6 +16,42 @@
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width">
 
+	<!-- AGLS Metadata -->
+	<link rel="schema.DCTERMS" href="http://purl.org/dc/terms/" />
+	<link rel="schema.AGLSTERMS" href="http://www.agls.gov.au/agls/terms/" />
+	<meta name="DCTERMS.title" content="Form page template - Australian Government Design System" />
+	<meta name="DCTERMS.creator" scheme="AGLSTERMS.AglsAgent" content="corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=designsystem@dta.gov.au" />
+	<meta name="DCTERMS.publisher" content="scheme=AGLSTERMS.AglsAgent; corporateName=Digital Transformation Agency, Australia; address=50 Marcus Clarke St Canberra ACT 2601; contact=designsystem@dta.gov.au" />
+	<meta name="DCTERMS.description" content="Form pages ask users a question and allow them to answer it" />
+	<meta name="DCTERMS.modified" content="scheme=DCTERMS.ISO8601; 2019-08-06T11:36:29+10:00" />
+	<meta name="DCTERMS.subject" content="design system, templates" />	
+	<meta name="DCTERMS.language" content="DCTERMS.RFC4646; en-AU" />
+	<meta name="DCTERMS.type" content="Form template" />
+	<meta name="DCTERMS.coverage" content="Australia" />
+	<meta name="DCTERMS.spatial" content="Australia" />
+	<meta name="AGLSTERMS.documentType" content="text" />
+	<meta name="msapplication-TileColor" content="#ffffff" />
+	<meta name="theme-color" content="#ffffff" />
+	<meta name="robots" content="index, follow" />
+	<meta name="author" content="Digital Transformation Agency" />
+	<meta name="description" content="Form pages ask users a question and allow them to answer it" />
+	
+	<!-- TWITTER CARD -->
+	<meta name="twitter:card" content="summary_large_image" />
+	<meta name="twitter:title" content="Australian Government Design System" />
+	<meta name="twitter:description" content="Form pages ask users a question and allow them to answer it" />
+	<meta name="twitter:site" content="@DTA" />
+	<meta name="twitter:image" content="https://designsystem.gov.au/assets/favicons/designsystem.jpg" />
+	
+	<!-- OPENGRAPH -->
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="Form page template - Australian Government Design System" />
+	<meta property="og:site_name" content="Australian Government Design System" />
+	<meta property="og:description" content="Form pages ask users a question and allow them to answer it" />
```

Comment:
I know this is what we have in designsystem.gov.au, but we could do something better here 😅

How about:
Forms are used to collect information from the end users with regards to a specific topic.